bbc_sale/data/action_automated_emails_magento.xml

```diff
@@ -7,7 +7,7 @@
         <record id="action_send_email_confirmation_sales_order_magento" model="ir.actions.server">
             <field name="name">Magento | Send Email After Confirming Sales Order</field>
             <field name="model_id" ref="sale.model_sale_order"/>
-            <field name="condition">object.create_uid.name == 'magento'</field>
+            <field name="condition">object.create_uid.name == 'Informatie over je bestelling | Order information | Babycare'</field>
```

Comment:
This looks like it can break again in the future. Can you check for object.wk_shop instead? Note that there are some orders with a wk_shop from other users. I believe that these could be copies from Magento orders and you might want to reset the wk_shop field in sale.order's copy() method.

Reply:
I see what you mean. This will work on the server actions with model_id = sale.model_sale_order. Unfortunately it does not work on server action 'action_send_email_validation_invoice_magento'. The model of this action is account.model_account_invoice and this model does not have wk_shop field (yet).
Of course it is possible to copy the wk_shop to the invoice model but that will be another PR from my side.

My proposal for now would be to give the condition based on the name (as my changes in the commit) and I will look into another way to make it more future-prove on a later time.

Or do you have another solution for now?

Reply:
Alternatively you could use `create_uid.login = 'magento'`

Reply:
Great solution!
I changed the conditions of the server actions and filters in my last commit.
I think it has to work like this.